oct/pages/base/base.py

```diff
@@ -15,16 +31,29 @@ def __init__(self, refference):
 
     contains = dict()
     """
+    contains class attribute contains page's static webelements. Webelements set
+    is the page specific.
+    Example:
     contains = {
         elementName: {
             "locator": (by, value),
             "class": Element
         },
     }
     """
+    loads = dict()
```

Comment:
If it's needed for loading the list of elements like product thumb the name of a variable can be 'contains_lists' or 'list_items' instead of 'loads'. Or it can be in contains:
```
contains = {
        elementName: {
            "locator": (by, value),
            "class": Element
        },
       "listElement": {
           "locator": (by, value),
           "class": ListElement,
           "is_list": True
        }
    }
```

Reply:
Ok, this is reasonable, going to apply changes